src/Resources/views/base_layout.html.twig

```diff
@@ -57,8 +57,8 @@ file that was distributed with this source code.
                             jQuery(document).ready(function() {
                                 Sonata.Page.init({
                                     url: {
-                                        block_save_position: '{{ sonata_admin.objectUrl('sonata.page.admin.page', 'edit', page) }}',
-                                        block_edit:          '{{ sonata_admin.url('sonata.page.admin.page|sonata.page.admin.block', 'edit', {'id': 'BLOCK_ID'}) }}'
+                                        block_save_position: '{{ sonata_page_admin.generateObjectUrl('edit', page) }}',
+                                        block_edit:          '{{ sonata_page_admin.generateUrl('sonata.page.admin.page|sonata.page.admin.block.edit', {'id': page.id, 'childId': 'BLOCK_ID'})
```

Comment:
I've been not able to test this, but given `'sonata.page.admin.page|sonata.page.admin.block.edit'` route contains a reference of `page` and `block`, I guess it should have `id` and `childId`, I'm not sure if this was working before.

If you can give it a try @core23 or someone using the bundle.

Reply:
I'm not using this bundle either, but should not we use `admin.idParameter` for these route parameters instead of having them hardcoded? Something like:
```twig
{% set sonata_page_block_admin = sonata_admin.adminPool
    .getAdminByAdminCode('sonata.page.admin.page|sonata.page.admin.block')
%}
{{
    sonata_page_block_admin.generateUrl(
        'edit',
        {
            (sonata_page_block_admin.idParameter): 'BLOCK_ID',
            (sonata_page_block_admin.parent.idParameter): sonata_page_block_admin.parent.id(sonata_page_block_admin.parent.subject) 
        }
    )
}}
```
BTW, the closing ` }}'` is missing in https://github.com/sonata-project/SonataPageBundle/pull/1297#discussion_r612153544.

Reply:
Oops thanks! we should probably use a twig linter 🤦 

I've changed the code a bit to not use `sonata_admin`.

Reply:
>Oops thanks! we should probably use a twig linter facepalm

Yes, I think we could catch syntax errors faster.

>I've changed the code a bit to not use sonata_admin.

That's great. Did you have chance to analyze my concern about the hardcoded route parameters?